packages/geocoding/lib/providers/geonames.provider.ts

```diff
@@ -0,0 +1,95 @@
+import { GeocodingResult } from "../model";
+import { Geometry } from "geojson";
+
+const baseUrl = "https://secure.geonames.org/searchJSON";
+
+/**
+ * Reference documentation: http://www.geonames.org/export/geonames-search.html
+ * @property lang Place name and country name will be returned in the specified language. Default is English.
+ * @property maxRows The maximal number of rows in the document returned by the service. Default is 100
+ * @property country Default is all countries. The country parameter may occur more than once, example: country=FR&country=GP
+ * @property style verbosity of returned xml document, default = MEDIUM
+ * @property type The format type of the returned document, default = xml
+ * @property username The parameter 'username' needs to be passed with each request. (https://www.geonames.org/login)
+ * @property east, west, north, south Restrict the search to the given bounding box
+ */
+
+export interface GeonamesOptions {
+    lang?: "de" | "fr" | "it" | "rm" | "en";
```

Comment:
the list of supported languages is wider and basically any iso 2-letter code can be provided as well as "local", see the doc:
![image](https://github.com/camptocamp/geospatial-sdk/assets/10629150/44b09caf-c715-4a5a-9992-a7d215472257)

So typing to `string` and mentioning in the doc that either "local" or a iso2 country code should be used would be perfect :)